internal/doctl/mylib.go

```diff
@@ -0,0 +1,182 @@
+package mylib
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/digitalocean/godo"
+	"github.com/pkg/errors"
+)
+
+//DoctlDependencies interface for doctl dependent functions
+type DoctlDependencies interface {
+	IsAuthenticated(token string) error
+	GetCurrentDeployment(appID string) ([]byte, error)
+	RetrieveActiveDeploymentID(appID string) (string, error)
+	RetrieveActiveDeployment(deploymentID string, appID string) ([]byte, error)
+	UpdateAppPlatformAppSpec(appID string) error
+	CreateDeployments(appID string) error
+	RetrieveFromDigitalocean() ([]byte, error)
+	RetrieveAppID(appName string) (string, error)
+	IsDeployed(appID string) error
+	ReDeploy(input string, appName string) error
+}
+
+//DoctlServices is a struct for holding doctl dependent function interface
+type DoctlServices struct {
+	Dep DoctlDependencies
```

Comment:
This probably shouldn't be in here.

Reply:
Can you help me or provide suggestion for cleaning this up. I used this format for mock function creation. But ended up not using mock testing. Should I create Empty struct for DoctlServices?